src/client/sandbox/timers.js

```diff
@@ -0,0 +1,112 @@
+import SandboxBase from './base';
+import nativeMethods from './native-methods';
+import { processScript } from '../../processing/script';
+import { isIE, version as browserVersion } from '../utils/browser';
+
+// NOTE: When you call the focus and blur function for some elements in IE, the event handlers  must be raised
+// asynchronously, but before executing functions that are called by using the window.setTimeout function. So,
+// we need to raise the handlers with a timeout, but do it before calling other asynchronous functions.
+export default class TimersSandbox extends SandboxBase {
+    constructor () {
+        super();
+
+        this.timeouts          = [];
+        this.deferredFunctions = [];
+        this.setTimeout        = nativeMethods.setTimeout;
+    }
+
+    _wrapTimeoutFunctionsArguments (args) {
+        var isScriptFirstArg = typeof args[0] === 'string';
+        var func             = !isScriptFirstArg ? args[0] : null;
+        var script           = isScriptFirstArg ? processScript(args[0]) : null;
+
+        if (isIE && browserVersion < 12) {
+            var timersSandbox = this;
+            var fnToRun       = isScriptFirstArg ? () => this.window.eval(script) : func;
```

Comment:
We should call `eval` here without context closure. So we need to use technique from `get-eval` instrumentation:
```js
() => {
  // Add comment here why we use this approach.
   var eval = window.eval;
   eval(script);
}
```